python/src/model/model.cpp

```diff
@@ -38,13 +73,23 @@ void add_binomial_to_module(py::module_& m) {
     using sgs_fixed_n_default_t =
         SimGlobalStateFixedNDefault<gen_t, value_t, uint_t, gr_t>;
     using kbs_fixed_n_default_t = KevlarBoundStateFixedNDefault<gr_t>;
+
     add_fixed_n_default<sgs_fixed_n_default_t, kbs_fixed_n_default_t>(m);
 
     using simple_selection_t = SimpleSelection<value_t>;
     add_simple_selection<simple_selection_t, gen_t, value_t, uint_t, gr_t>(m);
 
     using thompson_t = Thompson<value_t>;
     add_thompson<thompson_t, gen_t, value_t, uint_t, gr_t>(m);
+
+    {
```

Comment:
curious choice for extra scoping 🤔

Reply:
Oh that was remnant from an old version from before I pulled out the function. Removed!